src/rgw/rgw_log.h

```diff
@@ -95,11 +101,14 @@ struct rgw_log_entry {
     }
     if (struct_v >= 7) {
       ::decode(obj, p);
-    }
-    if (struct_v >= 8) {
-      ::decode(object_owner, p);
-      ::decode(bucket_owner, p);
-    }
+      if (struct_v >= 8) {
+	::decode(object_owner, p);
+	::decode(bucket_owner, p);
+	if (struct_v >= 9) {
```

Comment:
@mattbenjamin for code readability I'd prefer to avoid nested ifs like this, and with regular compile optimization it shouldn't affect execution time